components/dropbox/sources/all-updates/all-updates.mjs

```diff
@@ -34,9 +44,18 @@ module.exports = {
       await this.dropbox.initState(this);
     },
   },
-  async run(event) {
+  methods: {
+    getMeta(id, summary) {
+      return {
+        id,
+        summary,
+        tz: Date.now(),
+      };
+    },
+  },
```

Comment:
If you create a `common.mjs` file in sources, you can remove the `methods` and `hooks` sections.